charts/sorry-cypress/values.yaml

```diff
@@ -133,6 +133,8 @@ director:
     region: us-east-1
     accessKeyId: ""
     secretAccessKey: ""
+    acl: "public-read"
```

Comment:
Dunno if the default value should be an empty string (and then the default behavior would be implicit), or if it should be `public-read` (and then we are always overriding the default behavior, but IMHO it won't change in the future).